packages/expo-system-ui/ios/EXSystemUI.podspec

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'EXSystemUI'
```

Comment:
I'd start using `Expo` instead of `EX` — that's what I already did with `expo-modules-core`. So `EXSystemUI` -> `ExpoSystemUI` (also rename podspec file).

Reply:
Any preference on naming like `expo-system-ui`, which would be similar to expo-dev-client stuff?

Reply:
Imho it would look awkward to do `#import <expo-system-ui/*.h>` in ObjC or `import expo_system_ui` in Swift (afaik it doesn't allow hyphens) in case you want to import it in your native code (that was my main concern when renaming modules core). We could use podspec's `header_dir` to change it, but imho it makes more sense to keep the pod name and header name in sync 🤔 We can discuss it further tho